crypto/crypto.cpp

```diff
@@ -35,8 +35,24 @@ crypto::crypto(std::shared_ptr<bzn::options_base> options)
     }
 }
 
+const std::string&
+crypto::extract_payload(const bzn_envelope& msg)
+{
+    switch (msg.payload_case())
+    {
+        case bzn_envelope::kPbftRequest : return msg.pbft_request();
```

Comment:
This is not our coding standard. Each return should be contained withing a block below "case"